src/ThemeProvider.js

```diff
@@ -20,11 +19,10 @@ export const Base = styled.div`
   }
 `
 
-const ThemeProvider = ({ legacy, customBreakpoints, ...props }) => {
-  const baseTheme = legacy ? legacyTheme : nextTheme
-  const breakpoints = customBreakpoints || baseTheme.breakpoints
+const ThemeProvider = ({ customBreakpoints, ...props }) => {
+  const breakpoints = customBreakpoints || nextTheme.breakpoints
```

Comment:
Should we just rename this to `theme`?

Reply:
We are using theme as a const in the next line. I didn't know what to change its name to (maybe themeWithBreakpoints haha) so I kept the first imported theme as nextTheme.